doc/release-notes.md.in

```diff
@@ -64,11 +64,12 @@ AtomVM currently supports the following versions of ESP-IDF:
 
 | IDF SDK supported versions | AtomVM support |
 |------------------------------|----------------|
-| ESP-IDF [v4.4](https://docs.espressif.com/projects/esp-idf/en/v4.4.6/esp32/get-started/index.html) | ✅ |
-| ESP-IDF [v5.0](https://docs.espressif.com/projects/esp-idf/en/v5.0.4/esp32/get-started/index.html) | ✅ |
-| ESP-IDF [v5.1](https://docs.espressif.com/projects/esp-idf/en/v5.1.1/esp32/get-started/index.html) | ✅ |
+| ESP-IDF [v4.4](https://docs.espressif.com/projects/esp-idf/en/v4.4.7/esp32/get-started/index.html) | ✅ |
+| ESP-IDF [v5.0](https://docs.espressif.com/projects/esp-idf/en/v5.0.6/esp32/get-started/index.html) | ✅ |
+| ESP-IDF [v5.1](https://docs.espressif.com/projects/esp-idf/en/v5.1.3/esp32/get-started/index.html) | ✅ |
+| ESP-IDF [v5.2](https://docs.espressif.com/projects/esp-idf/en/v5.2.1/esp32/get-started/index.html) | ✅ |
 
-Building the AtomVM virtual machine for ESP32 is optional.  In most cases, you can simply download a release image from the AtomVM [release](https://github.com/atomvm/AtomVM/releases) repository.  If you wish to work on development of the VM or use one on the additional drivers that are available in the [AtomVM repositories](https://github.com/atomvm) you will to build AtomVM from source.  See the [Build Instructions](build-instructions.md) for information about how to build AtomVM from source code.
+Building the AtomVM virtual machine for ESP32 is optional.  In most cases, you can simply download a release image from the AtomVM [release](https://github.com/atomvm/AtomVM/releases) repository.  If you wish to work on development of the VM or use one on the additional drivers that are available in the [AtomVM repositories](https://github.com/atomvm) you will to build AtomVM from source.  See the [Build Instructions](build-instructions.md) for information about how to build AtomVM from source code.  Also, please check [`idf-version` on esp32-build.yaml](https://github.com/atomvm/AtomVM/blob/main/.github/workflows/esp32-build.yaml#L38) for the current versions including subminor number used in the CI build.
```

Comment:
I wonder if it might be better to point people to consult a recent workflow run from this link: https://github.com/atomvm/AtomVM/actions/workflows/esp32-build.yaml to check for patch release number that we are testing against, but in general we try to update out CI as soon as new releases are available. Just because any changes to the workflow are likely to change the line number of the version matrix, and it will be hard to remember to update this link when updating the workflow, since it isn't obvious that they are related.

Reply:
I think so; in fact, at first, I thought that link might be a good idea. But that link you showed does not yet show the version number, and users need to jump to one more link. Or it might be better just to remove the line number `#L38` from the link to the file `esp32-build.yaml`. Or perhaps simply indicate to users that you would use the latest subminor number (patch number).
In summary, which sentence do you think is best to include?

1. Also, please check [esp32-build.yaml](https://github.com/atomvm/AtomVM/actions/workflows/esp32-build.yaml) and the following link for the current versions including the subminor number used in the CI build and its status.
2. Also, please check [`idf-version` on esp32-build.yaml](https://github.com/atomvm/AtomVM/blob/main/.github/workflows/esp32-build.yaml) for the current versions including subminor number used in the CI build.
3. Also, we recommend you to use the latest subminor (patch) versions for source build.

Reply:
I think 3 is best, but it might be a good idea to include a link to the workflow for the exact version used for testing (as a disclaimer). We try to stay current, but it can take a few days from the time a new IDF version is released before we get an updated workflow merged into the CI.

So maybe, "We recommend you to use the latest subminor (patch) versions for source builds.  You can check the current version used for testing in the [esp32-build.yaml](https://github.com/atomvm/AtomVM/actions/workflows/esp32-build.yaml) workflow."

Reply:
Thank you for your suggestion! I added that sentence in 709eeb5b